x/liquidity/spec/06_end_block.md

```diff
@@ -1,23 +1,28 @@
-<!--
-order: 6
--->
+<!-- order: 6 -->
 
-# Before-End-Block
+ # Before-End-Block
```

Comment:
again, a general intro would be very helpful

Reply:
It is very general information in Cosmos-SDK concept.
People who are reading this tech spec is assumed to have enough knowledge on Cosmos-SDK structure.
If not, they have to read docs in Cosmos-SDK.
So, explaining common Cosmos-SDK concepts in this document is redundant.